README.md

```diff
@@ -102,15 +100,17 @@ Now, the contents of the `bindings/LDK` folder will have been completely regener
 ### Configuring Xcode for Framework Compilation
 Go to the `xcode` folder and open `LDKFramework.xcworkspace`.
 
-Then, navigate to the "LDKFramework" project and click on the LDKFramework target:
+Then, navigate to the "LDKFramework" project and click on the LDKFramework project target:
 ![](https://user-images.githubusercontent.com/5944973/175575527-97073a18-76fc-4ab0-928f-d40ac643e607.png)
 
-Search for "LDK_DIRECTORY":
-![](https://user-images.githubusercontent.com/5944973/175575621-38224096-4baa-44cc-8345-ec2b871fcbe6.png)
+Search for `LDK_C_BINDINGS_BASE`:
+![](https://user-images.githubusercontent.com/5944973/177420117-bcf7fc32-b07d-440f-84d1-fe2d373c355b.png)
+
+Here, enter the path: `${PROJECT_DIR}/../../bindings/artifacts/ldk-c-bindings`
```

Comment:
precisely the reason it can be parameterized. We won't even need a PROJECT_DIR in there

Reply:
we _could_ have these be the default values in the target settings, and this step will not have to be here at all and mentioned for those who want to point to their own directories to change it on their own. 

I personally don't see the issue if the point of the README is to get people to compile their frameworks to be able to copy and paste this in